admin/package.json

```diff
@@ -4,9 +4,9 @@
   "description": "GeoMapFish administration interface",
   "author": "",
   "dependencies": {
-    "bootstrap": "3.4.0",
+    "bootstrap": "4.5.0",
```

Comment:
@arnaud-morvan  Do you think that this change is OK?

Reply:
I'm pretty sure this will break near to everything on UI side.